cmd/buildah/images.go

```diff
@@ -253,7 +253,7 @@ func outputImages(ctx context.Context, systemContext *types.SystemContext, store
 		inspectedTime, digest, size, _ := getDateAndDigestAndSize(ctx, store, image)
 		if !inspectedTime.IsZero() {
 			if createdTime != inspectedTime {
-				logrus.Debugf("image record and configuration disagree on the image's creation time for %q, using the one from the configuration", image)
+				logrus.Debugf("image record and configuration disagree on the image's creation time for %q, using the one from the configuration", image.ID)
```

Comment:
it would probably be good to also include the inspectedTime as part of this output.
perhaps:  "%q, using the configuration time: %v", image.ID, inspectedTime"
(I think date/time and %v are happy)